### DISCUSSION TITLE
Automatically randomize Icebreaker question for team members

### DISCUSSION THREAD

**Discussion by gcrickman** [ParabolInc/parabol#5727] [Feedback]
**## Issue - Enhancement**

[A user](https://app.hubspot.com/contacts/3888472/contact/3254151) 🔒  writes:

> Quick feedback: last week for our retro, to make the social check-in more dynamic, we tried to refresh the question for each new person and it worked super great, it was more dynamic than the ones where everyone has the same question. We never thought about doing it but I think we will do that every time from now on.
> 
> Would be nice to have an option to make it automatic I think :)

### Acceptance Criteria (optional)
Users can:
   - Do this: Facilitator can set the icebreaker question to automatically randomize for each team member

**Comment by jordanh** [comment]
User can hit the refresh button. Iceboxing for now, will reopen if we hear this idea again

**Comment by jordanh** [comment]
Taking out of the icebox.

[A user](https://app.hubspot.com/contacts/3888472/contact/10423151) 🔒 writes:

> A "keep question random" button for the icebreaker would be cool. Something that automatically changes the question for each person as you go through the list at the beginning. Not sure what a nice workflow for that would look like though.

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/35899101) 🔒 says:

> I like to use a new icebreaker for each person. Is there a way you could make it easier to do that?

[a comment by an automated account is not shown]
